src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -99,7 +99,9 @@ void CreateInteractionChoiceSetRequest::Run() {
           .asInt();
 
   if (app->FindChoiceSet(choice_set_id_)) {
-    SDL_ERROR("Choice set with id " << choice_set_id_ << " is not found.");
+    SDL_ERROR("Choice set with id " << choice_set_id_
```

Comment:
@okozlovlux, 
Seem to be a Warning not an Error - please, see [Operational View -> Diagnostics](https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=280330801#SoftwareArchitectureDocument%28SAD%29-OperationalView)